src/remediations/remediations.queries.js

```diff
@@ -773,43 +773,20 @@ exports.getPlanSystemsDetails = async function (inventoryIds, chunkSize = 50) {
     return result;
 };
 
+// For now, we're returning an empty executors array because we no longer use playbook_run_executors or playbook_run_systems tables (receptor data)
```

Comment:
Something that might be useful for a lot of the queries that we're cleaning up...

We wouldn't need to make any API calls in `formatRHCRuns` for Direct or Satellite cases if we had a `dispatcher_runs_hosts` table. `dispatcher_runs_hosts` would store host-level execution status and counts that we get from Kafka Run Host events from playbook-dispatcher.. 
The `dispatcher_runs` table helps in a lot of cases but for endpoints like `GET /remediations/id/playbook_runs`, we need the host-level run data